dogpile/cache/region.py

```diff
@@ -1397,6 +1400,16 @@ def delete_multi(self, keys: Sequence[KeyType]) -> None:
 
         self.backend.delete_multi(keys)
 
+    def get_age(self, key: KeyType) -> float | None:
```

Comment:
we have not bumped our Python requirements yet and they are actually still on 3.6 (which is bad, but there it is), so this has to be `Optional[float]`.

Reply:
[done](https://github.com/sqlalchemy/dogpile.cache/pull/247/commits/dd3a26e2b52814e4a6d44249b500dfd3122abceb)